### PR TITLE
Update Railway from Hanh Dao

### PR DESCRIPTION
**Pull request by 4lm0nd**
Project Update

### CODE REVIEW

File: Utilities/railway/Utilities.java [hunk 1]
```diff
@@ -80,6 +84,15 @@ public void checkElementDoesNotExist(WebElement element) {
             System.out.println(e.getMessage());
         }
     }
-}
+
+    public String getDateLaterFromCurrentDate(int days) {
```
Comment (nbmhoang) [inline Utilities/railway/Utilities.java:88]: For any method that interact to Date/Time, my advice is use LocalDate(for date) and LocalDateTime(for date time). Its provide many method to do this

File: PageObjects/railway/GeneralPage.java [hunk 2]
```diff
@@ -82,6 +85,10 @@ public int getTableRow(String table, String departStation, String arriveStation,
         }
         return 0;
     }
+
+    public void scrollToFindElement(WebElement element) {
+        element.sendKeys(Keys.PAGE_DOWN);
```
Comment (nbmhoang) [inline PageObjects/railway/GeneralPage.java:90]: Why we use this key here?

File: Utilities/railway/Utilities.java (hunk 1 above)
Comment (nbmhoang) [reply Utilities/railway/Utilities.java:88]: Also, method come from this class should be declared as **static**

Comment (4lm0nd) [reply Utilities/railway/Utilities.java:88]: update changes

File: PageObjects/railway/GeneralPage.java (hunk 2 above)
Comment (4lm0nd) [reply PageObjects/railway/GeneralPage.java:90]: press page down button on the keyboard cho scroll page down

Comment (nbmhoang) [reply PageObjects/railway/GeneralPage.java:90]: Could we use Javascript here?

Comment (4lm0nd) [reply PageObjects/railway/GeneralPage.java:89]: yes, but why we should use javascrip instead of key page down?

Comment (nbmhoang) [reply PageObjects/railway/GeneralPage.java:89]: It will same as users behavior(users scroll by mouse, not press key)

Comment (4lm0nd) [reply PageObjects/railway/GeneralPage.java]: update changes :)))

**Review by nbmhoang — CHANGES_REQUESTED** [review]
Exception Constants/DriverManager class, all properties inside class/test case must not be **static**

**Review by nbmhoang — CHANGES_REQUESTED** [review]
Rename data object class name and use correct constructor/getter/setter

File: DataObject/railway/BookTicket.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
+package railway;
+
+public class BookTicket {
+    private String DepartDate;
+    private String DepartStation;
+    private String ArriveStation;
+    private String SeatType;
+    private String TicketAmount;
```
[the remark this answers is not in the document]
Comment (4lm0nd) [reply DataObject/railway/BookTicket.java:8]: "Ticket amount" field is a combobox. I use select item by text

**Pull request merged**